poetry.lock

```diff
@@ -15,9 +15,9 @@ optional = false
 python-versions = ">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*"
 
 [package.extras]
-dev = ["coverage[toml] (>=5.0.2)", "hypothesis", "pympler", "pytest (>=4.3.0)", "six", "mypy", "pytest-mypy-plugins", "zope.interface", "furo", "sphinx", "sphinx-notfound-page", "pre-commit", "cloudpickle"]
-docs = ["furo", "sphinx", "zope.interface", "sphinx-notfound-page"]
```

Comment:
Hmm, 🤔 I used `poetry@master` (not e.g. 1.1.13) to bump the `dulwich` dependency. Was that a bad move?

I can see these delimiters being changed from `.` to `-` and tests seem to fail because of it.

Reply:
Yup, worked better. Resolving this thread.